github/ci/prow-deploy/kustom/base/configs/current/orgs/orgs.yaml

```
@@ -659,6 +661,13 @@ orgs:
           - aglitke
         repos:
           containerized-data-importer-api: admin
+      community-health-files:
```

nit: in general we add something to distinguish teams from the repo name itself, helps in reading, here I'd suggest using maintainers.

```suggestion
      community-health-files-maintainers:
```

---

ok, got it.